Satscard Integration (Android)

PeteClubSeven opened the pull request:
Hey, as part of the Satscard bounty these are the strings I added. I believe @kingonly wanted to review these and make some modifications

Note: 60e316b3471e2b4464b33307d762218b5beb40bf will stop future problems where generating the translation texts causes English to no longer be the default locale when a users device is set to an unsupported locale.

kingonly commented:
@PeteClubSeven I sent a few changes via email

PeteClubSeven commented:
> @PeteClubSeven I sent a few changes via email

Done, let me know if you want any further changes

ubbabeck reviewed (approved):
Thanks looks good to me.

Why did you specify android, is there a reason for us to need alternaltive translations for iOS?

ademar111190 commented on l10n.yaml:
```diff
@@ -3,5 +3,6 @@ template-arb-file: app_en.arb
 output-localization-file: breez_translations.dart
 output-class: BreezTranslations
 output-dir: lib/generated
+preferred-supported-locales: ["en"]
```
What is the function of this change?

ademar111190 replied:
Ok; I just realized that is the change you mentioned that is fixing the English as first in the languages order; that is amazing; thank you very much :rocket:

ademar111190 commented on test/breez_translations_test.dart:
```diff
@@ -906,6 +906,102 @@ void main() {
         expect(text.contains(maxSats), true);
       });
+
+      test("satscard_error_invalid_body for ${locale.locale}", () {
```
Thanks for adding the tests :pray:

PeteClubSeven replied:
If there's anything else missing let me know, I tried to follow the standard practices of your app but it's a large codebase so I likely missed some things ❤️

ademar111190 commented:
@kingonly are the changes addressed?

PeteClubSeven commented:
@ademar111190 @ubbabeck I added one additional dialog because I replaced the auto-closing "Broadcast Transaction" page with a pop up when the Satscard is successfully swept

ademar111190 commented:
I'm merging the PR.

If there are changes from @kingonly not addressed yet we can open a new PR later.
